src/routes/transactions/entities/queued-items/transaction-group.entity.ts

```diff
@@ -0,0 +1,6 @@
+import { MultisigTransaction } from '../../../../domain/safe/entities/multisig-transaction.entity';
+
+export class TransactionGroup {
```

Comment:
This is used only in the `QueuedItemsMapper`.

I would just have the class there and not export it to increase encapsulation.